CMakeLists.txt

```diff
@@ -249,6 +249,7 @@ set(MODULES
     src/Options.cpp
     src/OptionHandler.cpp
     src/ProgressReporter.cpp
+    src/RawAudioFileReader.cpp
```

Comment:
I don't think we need to add a new class, as we can change the existing `SndFileAudioFileReader`.

Reply:
I see :+1: